adm_program/modules/members/members_data.php

```diff
@@ -328,20 +328,9 @@
         }
     }
 
-    if(strlen($row['email']) > 0)
-    {
-        if($gPreferences['enable_mail_module'] != 1)
-        {
-            $mailLink = 'mailto:'.$row['email'];
-        }
-        else
-        {
-            $mailLink = ADMIDIO_URL.FOLDER_MODULES.'/messages/messages_write.php?usr_id='.$row['usr_id'];
-        }
-
-        $userAdministration .= '<a class="admidio-icon-link" href="'.$mailLink.'"><img src="'.THEME_URL.'/icons/email.png"
```

Comment:
is it useful?

I thought to much buttons are bad but if you @thomas-rcv @sirnone  also think we shouldn't remove the email link in the user management than we don't